dev/tools/backport-pr.sh

```diff
@@ -30,6 +53,14 @@ MESSAGE=$(git log master --grep "Merge PR #${PRNUM}" --format="%s" | sed 's/Merg
 if git checkout -b "${BRANCH}"; then
 
     if ! git cherry-pick -x "${RANGE}"; then
+        if [[ "$NO_CONFLICTS" == "true" ]]; then
+            git status
+            echo "Conflicts! Aborting..."
+            git cherry-pick --abort
+            git checkout -
```

Comment:
Shouldn't this be `checkout --`?

Reply:
with -f too maybe
Test it manually until it works I guess.

Reply:
It was tested manually ;-)
`git checkout -` is supposed to checkout back to the previous ref. AFAIK `git checkout --` (possibly with `-f`) would rather be about undoing partial changes, but this is already being taken care of by `git cherry-pick --abort`.